users/views.py

```diff
@@ -137,3 +138,32 @@ def post(self, request):
             return JsonResponse({'MESSAGE': 'KEY_ERROR'}, status=400)
         except TypeError:
             return JsonResponse({'MESSAGE': 'TYPE_ERROR'}, status=400)
+
+class UserProductLikeView(View):
+    @utils.login_required
+    def post(self, request):
+        try:
+            data    = json.loads(request.body)
+            user    = getattr(request,'user',None)
+            product = Product.objects.get(id=data['product'])
+            like   = UserProductLike.objects.filter(user=user, product=product).first()
+
+            if like:
+                like.is_like = not like.is_like
+                like.save()
+                
+                return JsonResponse({'LIKE': like.is_like}, status=200)
+            UserProductLike.objects.create(user=user, product=product, is_like=True)
+
+            return JsonResponse({'LIKE': True}, status=200)
+        except json.decoder.JSONDecodeError:
```

Comment:
```suggestion

            if like:
                like.is_like = not like.is_like
                like.save()
                return JsonResponse({'LIKE': like.is_like}, status=200)
            
            UserProductLike.objects.create(user=user, product=product, is_like=True)
            return JsonResponse({'LIKE': True}, status=200)
        except json.decoder.JSONDecodeError:
```
논리상 띄어쓰기가 달라지는 것이 좋을 것 같습니다